src/prebid.js

```diff
@@ -70,6 +70,11 @@ function setRenderSize(doc, width, height) {
 }
 
 export const checkAdUnitSetup = hook('sync', function (adUnits) {
+  let adUnitsLen = adUnits.length;
+  if (adUnitsLen > 15) {
+    utils.logInfo(`Current auction contains ${adUnitsLen} adUnits.`);
```

Comment:
Should we add auction id in the message as well ? It will help them track down everything related to auction.

Reply:
I updated the log message to include some additional info.  Please take a look.